components/function-controller/README.md

```diff
@@ -102,19 +113,25 @@ Follow these steps to prepare the environment you will use to deploy the Control
 
 ### Deploy the controller
 
-1. Use the `make` targets to build the Function Controller image, tag it to the value of the **IMG** environment variable, and push it to the remote container registry.
+To deploy the Function Controller to the `serverless-system` Namespace run
 
-    ```bash
-    make docker-build
-    make docker-push
-    ```
+```bash
+make deploy
+```
 
-2. After the image has been successfully pushed, deploy the Function Controller to the `serverless-system` Namespace.
+This runs `ko` to build your image and push it to your configured docker repository. 
+See [google/ko](https://github.com/google/ko) for more details about `ko`
 
-    ```bash
-    make deploy
-    ```
+`ko` requires the environment variable `$KO_DOCKER_REPO` to point to a valid docker repository
+
+```bash
+#for hub.docker.io
+export KO_DOCKER_REPO=<your dockerhub username>
 
+#to use your local docker daemon and its image store
+export KO_DOCKER_REPO=ko.local
+
+```
```

Comment:
@bszwarc really insisted on not touching this numbering and indentation.

Reply:
That is true, but as @k15r  reworked this section it no longer includes two steps. If it's just one step with some explanation, we can skip the numbering.

Reply:
Correct, my bad